app/groceries/grocery-list/item-status.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { Grocery } from "../shared";
```

Comment:
Technically there should probably be an empty line between these two imports, I think.